File: HahaKoreaShop/app/src/main/java/com/ncggloval/hahakoreashop/common/HNApplication.java
package com.ncggloval.hahakoreashop.common;

//import com.tencent.mm.sdk.openapi.IWXAPI;

import com.tencent.mm.sdk.openapi.IWXAPI;

import java.net.CookieManager;

/**
 * Created by skcrackers on 5/26/16.
 */
public class HNApplication {
    public static boolean mSigned = true;
    public static String URL = "https://hahakoreashop.com";
    public static IWXAPI mWechatApi;
    public static String mWechatToken = "";
    public static String mWechatUserInfo = "";
    public static String mDeviceId = "";

    public static boolean mIsFirstLoading = true;

    // wechat
    public static final String APP_ID = "wx5cab472ee70264e7";
    public static final String APP_SECRET = "12c372a4c24f3ae30bc2a642b53c63a8";

    // baidu
    public static final String API_KEY = "api_key";

    public static CookieManager mCookieManager;

    public static CookieManager getCookieManager() {
        if (mCookieManager == null) {
            mCookieManager = new CookieManager();
        }

        return mCookieManager;
    }
}

File: 2022/HahaKoreaShop/app/src/main/java/com/ncggloval/hahakoreashop/common/HNApplication.kt
package com.ncggloval.hahakoreashop.common

import com.tencent.mm.sdk.openapi.IWXAPI
import java.net.CookieManager

//import com.tencent.mm.sdk.openapi.IWXAPI;
/**
 * Created by skcrackers on 5/26/16.
 */
object HNApplication {
    var mSigned = true
    var URL = "https://hahakoreashop.com"
    var PUSH_URL = URL + "/m/app/pushRegister.asp"
    var mWechatApi: IWXAPI? = null
    var mWechatToken = ""
    var mWechatUserInfo = ""
    var mDeviceId = ""
    var mIsFirstLoading = true

    // wechat
    const val APP_ID = "wx5cab472ee70264e7"
    const val APP_SECRET = "12c372a4c24f3ae30bc2a642b53c63a8"

    // baidu
    const val API_KEY = "api_key"
    var mCookieManager: CookieManager? = null
    val cookieManager: CookieManager?
        get() {
            if (mCookieManager == null) {
                mCookieManager = CookieManager()
            }
            return mCookieManager
        }
}